src/main/resources/application.properties

```diff
@@ -3,8 +3,13 @@ logging.level.org.atmosphere = error
 spring.mustache.check-template-location = false
 
 vaadin.launch-browser=false
-vaadin.whitelisted-packages = com.vaadin,org.vaadin,dev.hilla,com.vaadin.hackathon
+vaadin.allowed-packages = com.vaadin,org.vaadin,dev.hilla,com.vaadin.hackathon
 spring.jpa.defer-datasource-initialization = true
 
 github.release.api.url = https://api.github.com/repos/vaadin/platform/releases/tags/%s
-github.repository.path = /Users/manolo/Github/platform/platform
+github.repository.owner = vaadin
+github.repository.name = platform
+
+tagservice.tagcache.enabled = true
+#max age of entries in seconds before the API gets queried again
+tagservice.tagcache.maxAge = 3600
```

Comment:
add this and use in code:

```
tagservice.tagcache.folder=${CACHEFOLDER:${TMPDIR:/tmp}}
github.personal.token = ${GITHUB_TOKEN}
```

Reply:
That path will break under Windows. I use `${CACHEFOLDER:${TMPDIR:${TMP}}}` instead.